source/material_model/melt_global.cc

```diff
@@ -396,6 +398,25 @@ namespace aspect
                              "Whether to include melting and freezing (according to a simplified "
                              "linear melting approximation in the model (if true), or not (if "
                              "false).");
+          prm.declare_entry ("Melting rate for operator splitting", "1e-3",
+                             Patterns::Double (0),
+                             "In case the operator splitting scheme is used, the porosity field can not "
+                             "be set to a new equilibrium melt fraction instantly, but the model has to "
+                             "provide a melting rate instead. This rate defines how fast melting happens, "
+                             "more specifically, in the time that is defined by the inverse value of the "
+                             "melting rate, the deviation of the porosity from the equilibrium melt "
+                             "fraction will be reduced to a fraction of $1/e$. So if the melting rate is "
+                             "large (if the inverse value of the melting rate is small) compared to the "
+                             "time step size, the reaction will be so fast that the porosity is very close "
+                             "to the equilibrium melt fraction after reactions are computed."
+                             "\n\n"
+                             "Also note that the time step defined by the inverse of this melting rate has "
+                             "to be larger or equal to the reaction time step used in the operator splitting "
+                             "scheme, otherwise reactions can not be computed. If the model does not use "
+                             "operator splitting, this parameter is not used. "
+                             "If this parameter is set to 0, no melting and freezing occurs in the model. "
```

Comment:
Ah, is this the reason you want to use the inverse? Is there no better way to allow for no freezing / no melting?

Reply:
You could still set the time scale to 1e100 and have essentially no melting/freezing (and this is something that only comes up in test cases anyway), so I guess this is fine.